source/base/parameter_handler.cc

```diff
@@ -278,34 +318,90 @@ namespace Patterns
-
-    if (lower_bound <= upper_bound)
+    switch (style)
+      {
+      case Machine:
       {
-        // bounds are valid
-        description << description_init << " ";
-        // We really want to compare with ==, but -Wfloat-equal would create
-        // a warning here, so work around it.
-        if (0==std::memcmp(&lower_bound, &min_double_value, sizeof(lower_bound)))
-          description << "-MAX_DOUBLE";
+        std::ostringstream description;
+
+        if (lower_bound <= upper_bound)
+          {
+            // bounds are valid
+            description << description_init << " ";
+            // We really want to compare with ==, but -Wfloat-equal would create
+            // a warning here, so work around it.
+            if (0==std::memcmp(&lower_bound, &min_double_value, sizeof(lower_bound)))
+              description << "-MAX_DOUBLE";
+            else
+              description << lower_bound;
+            description << "...";
+            if (0==std::memcmp(&upper_bound, &max_double_value, sizeof(upper_bound)))
+              description << "MAX_DOUBLE";
+            else
+              description << upper_bound;
+            description << " (inclusive)]";
+            return description.str();
+          }
         else
-          description << lower_bound;
-        description << "...";
-        if (0==std::memcmp(&upper_bound, &max_double_value, sizeof(upper_bound)))
-          description << "MAX_DOUBLE";
+          {
+            // invalid bounds, assume unbounded double:
+            description << description_init << "]";
+            return description.str();
+          }
+      }
+      case Text:
+      {
+        if (lower_bound <= upper_bound)
+          {
+            std::ostringstream description;
+
+            description << "a floating point number v such that ";
+            if (0==std::memcmp(&lower_bound, &min_double_value, sizeof(lower_bound)))
+              description << "-MAX_DOUBLE";
+            else
+              description << lower_bound;
+            description << " <= n <= ";
+            if (0==std::memcmp(&upper_bound, &max_double_value, sizeof(upper_bound)))
+              description << "MAX_DOUBLE";
+            else
+              description << upper_bound;
+
+            return description.str();
+          }
         else
-          description << upper_bound;
-        description << " (inclusive)]";
-        return description.str();
+          return "a floating point number";
       }
-    else
+      case LaTeX:
       {
-        // invalid bounds, assume unbounded double:
-        description << description_init << "]";
-        return description.str();
+        if (lower_bound <= upper_bound)
+          {
+            std::ostringstream description;
+
+            description << "a floating point number $v$ such that $";
+            if (0==std::memcmp(&lower_bound, &min_double_value, sizeof(lower_bound)))
+              description << "-\\text{MAX_DOUBLE}";
```

Comment:
Can you try and make sure that we don't get a latex error on using `$-\text{MAX_DOUBLE}$`? That is, is latex ok with the bare underscore, or does it need `MAX\\_DOUBLE` here?